package.json

```diff
@@ -50,17 +58,22 @@
     "prettier-check": "^2.0.0",
     "rimraf": "2.6.2",
     "semantic-release": "^11.0.2",
+    "source-map-support": "^0.5.0",
     "tslint": "^5.4.2",
+    "tslint-config-prettier": "^1.6.0",
     "typescript": "^2.2.1"
   },
   "dependencies": {
     "babel-core": "^6.26.0",
     "babel-preset-env": "^1.6.1",
     "babel-register": "^6.26.0",
     "glob": "^7.1.2",
+    "got": "^8.0.1",
     "mz": "^2.7.0",
     "recast": "^0.13.0",
-    "resolve": "^1.5.0"
+    "resolve": "^1.5.0",
+    "tmp": "^0.0.33",
+    "whatwg-url": "^6.4.0"
```

Comment:
I'm using this instead of the builtin `url` since node v6.x doesn't have `url.URL`.

Reply:
Tests pass now!